.travis/opensuse.sh

```diff
@@ -7,6 +7,8 @@ ${DOCKER_EXEC} zypper -n install time.x86_64
 ${DOCKER_EXEC} zypper -n ref
 ${DOCKER_EXEC} zypper -n install rpmdevtools
 ${DOCKER_EXEC} rpmdev-setuptree
+${DOCKER_EXEC} zypper -n ar -f -G http://download.opensuse.org/repositories/Virtualization/${OS_NAME_VERSION}/Virtualization.repo
```

Comment:
I'm a bit confused here. For OpenSUSE it looks like Tumbleweed is the first release to support hwloc-1.11. Older releases provide older versions. For SLE12 it looks like they are already up to hwloc-2.0. Don't we have to support versions of hwloc both before and after 1.11 in order to support a variety of SUSE platforms? That means the PBS Pro code that parses the output needs to be tolerant of both "Package" and "Socket".